core/misc/backdrop.js

```diff
@@ -446,6 +446,60 @@ Backdrop.ajaxError = function (xmlhttp, uri, customMessage) {
   return message;
 };
 
+/**
+ * Check to see when webfont has loaded and adjust the tabs display
+ * @param fontName
+ *   Font name as shown in CSS
+ * @param callback
+ *   Function to run once font has loaded
+ */
```

Comment:
As this is a new feature. Let's add a line for `@since 1.5.0` indicating the version this function was added.